onboaring.md

```diff
@@ -0,0 +1,345 @@
+
+#### Warehouse as publish target
+
+By default, `npm` uses the global `registry` setting as endpoint to `publish` modules. This endpoint should be changed locally in your module to point to Warehouse. The local settings will override the global configuration.
+
+Add the following property to your `package.json`. It should be a top-level property, e.g. at the level of `main`, `dependencies`, etc. [More details on npm configuration can be found here][npm-config].
+
+```js
+"publishConfig": {
+  "registry": "https://wherever-you-deployed-warehouse.ai"
+}
+```
+
+#### Setup `wrhs.toml`
+
+Create a new file in the root folder named `./wrhs.toml`. This configuration file is read by Warehouse to understand
+additional build details. This file follows the `toml` format, which is [described in detail here][toml].
+The file describes build process outcomes. Defining what `files` should be uploaded to CDN, how minification
+should be done, etc.
+
+To get started, indicate which files should be uploaded to CDN. The filenames should be changed to match the output
+from your build. The list acts as a filter of assets. By default Warehouse will read all files from
+the output directory of your build. This ensures only the files that you specify will be uploaded to CDN.
+In the example Warehouse expects to find asset files in the folder `./dist`, named `output[.min].js` or
+`output[.min].css` respectively.
+
+```ini
+[files]
+dev = ['dist/ouput.js', 'dist/output.css']
+test = ['dist/ouput.js', 'dist/output.css']
+prod = ['dist/ouput.min.js', 'dist/output.min.css']
+```
+> We plan to do all configuration from this file in the future.
+> This will allow us to expose more features and make setup easier.
+
+From this example you can discern that uploaded files are configurable per environment and file paths are
+determined relative to the root of your module. The `dist` directory is not an arbitrary choice, `warehouse.ai` will need the `dist` directory to explicitly exist so it knows which files to serve.
+
+Finally, if you are using `webpack`, you will need a `webpack.config.js` in the root directory. Make sure that files in your `wrhs.toml` match the ouput of your `webpack.config.js`. It is important to note that all `warehouse.ai` is doing is to call `webpack` in this case, thus all of your configuration must live within this file (not as command line arguments). 
+
+```js
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'output.js'
+  }
+};
+```
+#### Extract Config
+
+COMING SOON: You will be able to extract your warehouse.ai configuration to keep it in one place by using [extract-config]. 
+
+### 4. Publishing your module
```

Comment:
I feel like at this point (at the end of the last step), this document is done and can just point to other documents. something like

```md
That's it, your package is setup to use `warehouse.ai`.

You can now publish your package with `npm publish` as you normally would and use npm dist-tags to promote your build through environments. See [common commands][new doc here] for help with how to do that.

There are also more [advanced configuration options][another new doc] that allow you to control minification, create dependent builds, create locale-specific builds, etc.

You can also use [`warehouse.ai-api-client`] in your application to consume the [build output][more new doc here].
```

The new docs can just be the content that you have below, just organized into smaller, more targeted docs.

Reply:
oops, missed this comment in the first round, making requested changes for it now.